libnmstate/netapplier.py

```diff
@@ -201,6 +212,42 @@ def _verify_change(desired_state):
     desired_state.verify_dns(current_state)
 
 
+def _prepare_vlan_ifaces_for_mtu_change(desired_state):
+    current_state = state.State(netinfo.show())
+    current_ifaces = list(six.viewvalues(current_state.interfaces))
```

Comment:
won't it be more efficient to use `six.viewitems` instead ? 

That way, you'd avoid getting the ifstate['name'] in line https://github.com/nmstate/nmstate/pull/563/files#diff-6b531a9cd456f54d73ad106d72174513R226 - if you unpack both ifname, ifstate on the for at line https://github.com/nmstate/nmstate/pull/563/files#diff-6b531a9cd456f54d73ad106d72174513R223. 

just a nit.

Reply:
I missed that, thanks!